src/pages/api/user/ticks.ts

```diff
@@ -60,12 +79,12 @@ const handler: NextApiHandler<any> = async (req, res) => {
         ret.forEach((tick) => {
           const newTick: Tick = {
             name: tick.Route,
-            notes: tick.notes,
```

Comment:
This is a bug -- the CSV I downloaded uses "Notes":
<img width="558" alt="image" src="https://github.com/OpenBeta/open-tacos/assets/3641356/1b9905f5-e085-4bff-956f-fcaf69508f6e">